### DISCUSSION TITLE
Total de PMs da classe Druida

### DISCUSSION THREAD

**Discussion by malkavfelipe** [YuriAlessandro/gerador-ficha-tormenta20#224] [Problemas]
Gerando algumas fichas, notei que a classe Druida não está somando o modificador de atributo no total de PMs.
E no livro está assim:
"Seu atributo-chave para lançar magias é Sabedoria e você soma seu bônus de Sabedoria no seu total de PM".
Mais alguém se separou com esse erro?

**Comment by isaacbatst** [comment]
Obrigado por reportar, a gente realmente não percebeu esse detalhe. Assim que possível vamos corrigir.

**Comment by YuriAlessandro** [comment]
@isaacbatst já colocou como uma issue (#225) e assim que estiver no ar eu aviso aqui e fecho o tópico. Obrigado mais uma vez pelo feedback.

**Comment by YuriAlessandro** [comment]
Isso já foi corrigido. Estou fechando o tópico.
